lib/skate_web/router.ex

```diff
@@ -159,9 +159,10 @@ defmodule SkateWeb.Router do
     get "/location_search/place/:id", LocationSearchController, :get
     get "/location_search/search", LocationSearchController, :search
     get "/location_search/suggest", LocationSearchController, :suggest
+    get "/detours", DetoursController, :detours
+    get "/detours/:detour_id", DetoursController, :detour
```

Comment:
I think possibly the addition of this endpoint snuck in ahead of the associated controller change? 

This (and the associated change in `api.ts`) I would recommend yanking from this PR and moving into https://github.com/mbta/skate/pull/2795

Reply:
That would balloon this PR quite a bit I think, so I kept the controller change out of this PR by choice, given nowhere in the UI references this route, and it doesn't cause any test failures

Reply:
I did notice that it didn't seem to be breaking anything, which is why I approved. 

I think what I was suggesting would shrink this PR, not balloon it?

Reply:
Ohhhhhhh I misread! Yank these changes from this PR to #2795, got it. I don't quite understand how to cherry pick lines in a file and commit some of those line changes to one branch but other line changes to another branch. Like, most of the changes in `api.ts` should be here, but the two route changes should be in the other PR. Thoughts @firestack?

Reply:
If you could get it into the other PR that'd be preferred but it's not in the user flow yet so I think it's fine.